src/app/home/event-user-list/event-user-list.component.html

```diff
@@ -0,0 +1,25 @@
+<clr-modal #userListModal [(clrModalOpen)]="open" [clrModalSize]="'lg'">
+    <h3 class="modal-title">List of all registered users</h3>
+    <div class="modal-body">
+        <table class="table table-compact">
+            <thead>
+                <tr>
+                    <th>User</th>
+                    <th>Number of Sessions</th>
+                    <th>Show Sessions</th>
+                </tr>
+            </thead>
+            <tbody>                
+                <tr *ngFor="let u of users">
+                    <td>{{u.email}}</td>
+                    <td>{{getUsersProgress(u)}}</td>
+                    <td (click)="listFilterName(u.email)"><clr-icon shape="child-arrow"></clr-icon></td>
+                </tr>                
+            </tbody>
+        </table>
+        
+    </div>
+    <div class="modal-footer">
+        <button class="btn btn-outline" (click)="open = false">Close</button>
```

Comment:
```suggestion
        <button class="btn btn-outline" (click)="this.userListModal.close()">Close</button>
```

I think if we use the `close` method here, then we can get rid of the `open` and `userListOpen` properties and all related bindings.